contrib/docker-compose/standalone/haproxy/etc/haproxy.cfg.tmpl

```diff
@@ -144,7 +144,30 @@ frontend fe_http_handler
 
     mode http
 
-    # Instance is not stopping
+    # ~~ HAProxy LB Configuration ~~
+    # Enable/Disable trust of Cloudflare and/or X-Forwarded-For headers
+    #
+    # By default HAProxy is running at the edge of your infrastructure
+    # and picks the client IP from the TCP connection. If you are running
+    # HAProxy behind a reverse proxy or Cloudflare, you need to enable
+    # the following options to trust the headers.
+
+    # HAProxy behind Cloudflare
+    #http-request set-var(txn.cloudflare) str(ENABLED)
+
+    # HAProxy behind a reverse proxy
+    #http-request set-var(txn.cdn) str(ENABLED)
+
+    # HAProxy behind Cloudflare and/or a reverse proxy
+    acl is_behind_cloudflare var(txn.cloudflare) -m found
+    acl is_behind_proxy      var(txn.cdn) -m found
+
+    # ~~ End HAProxy LB Configuration ~~
+
+    # Host header is set
+    acl is_host_set hdr(host) -m found
+
+    # Instance is stopping
```

Comment:
Reversed boolean?

Reply:
It's cosmetic fix. Comment didn't correspond to implementation: `stopping eq 1`